HSDP-1D.md

```diff
@@ -0,0 +1,89 @@
+```
+                              __
+                 ____________ ____
+             _______________  _________
+          _________________  ______________
+       ___________________  __________________                 __________________
+     _____________________  ________________________        ________________________
+    _______________________  ______________    _____          _______      __________
+   _________________________  ________________     __    ________________    ________
+  ___________________________ ____         _______      _______    ________   _______
+  ____________________        _                ___        ________    _____    ______
+  _________________                                        _________   _____   ______
+  ________________                                         _________   _____   ______
+  ________________                                       ___________   ____   ______
+  ________________                                   ______________    ____  ______
+  _________________                              _________________    ___   _____
+   ____________________                    ______________________    __   ____
+    __________________________________________________________
+     _______________________________________________________
+       __________________________________________________
+         ____________________________________________
+            ____________________________________
+                 _______________________
+
+-----------------------------------------------------------------------------------------
+Hardware Info : 
+-----------------------------------------------------------------------------------------
+Device Name       : High Speed Data Printer Model 1Data (HSDP-1D)
+Device ID         : TBD
+Device Type       : TBD
+Manufacturer Name : SlapDragon
+Manufacturer ID   : 0xF6976D00
+
+-----------------------------------------------------------------------------------------
+Description :
+-----------------------------------------------------------------------------------------
+The HSDP line is all about printing as fast as possible the data you need.
+
+The HSDP offer you the best way to store you data in term of cost efficency.
+Paper power and data is all you need to run a HSDP, its clever design combining the 
+speed of the line matrix and the lack of nedd for ink of a spark printer design will 
+really cut the cost of your radio survey by orders of magnitude.
+
+The HSDP line is also very quiet, SlapDragon guarantee less than 95dB of noise at full 
+speed !
+
+The HSDP-1D is a Data version of the model 1, equiped with curring edge memory chips 
+and buses this printer will be able to log everything you throw at it.
+It is built to print large amount of data in a smaller amount of time and lack 
+typesettings possibilities of the Graph variant.
```

Comment:
"and lack typesettings possibilities" -> "and lacks the typesetting possibilities"